Formula/m/mockery.rb

```diff
@@ -1,8 +1,8 @@
 class Mockery < Formula
   desc "Mock code autogenerator for Golang"
   homepage "https://github.com/vektra/mockery"
-  url "https://github.com/vektra/mockery/archive/refs/tags/v2.53.3.tar.gz"
-  sha256 "c9495d19de1ca52a61a9ed299d9988a4f4a3cf1ad614954a15ca3920d8b852a2"
+  url "https://github.com/vektra/mockery/archive/refs/tags/v3.0.0.tar.gz"
+  sha256 "472380b8d1b0838274e3e3f8f11008504840aa08ffe61ce3184804d29f111b29"
   license "BSD-3-Clause"
   head "https://github.com/vektra/mockery.git", branch: "master"
```

Comment:
"`v3`" branch is not yet set as the default one, but I expect it to become the default one soon
```suggestion
  head "https://github.com/vektra/mockery.git", branch: "v3"
```
https://go.dev/doc/modules/major-version

<img width="871" alt="image" src="https://github.com/user-attachments/assets/018fbe9e-9309-400a-930c-7266abf10a39" />

Current `master` will _likely_ be renamed to `v2`.